react/components/AutocompleteInput/SearchInput/index.tsx

```diff
@@ -24,10 +24,13 @@ const propTypes = {
   onBlur: PropTypes.func,
   /** Determine if the input and the button should be disabled */
   disabled: PropTypes.bool,
+  /** Determine the search bar size */
+  barSize: PropTypes.string,
```

Comment:
shouldn't this be `size`?

Reply:
I explained why I used `barSize` instead [here](https://github.com/vtex/styleguide/pull/1251#discussion_r441763471)

Reply:
This will do it
```ts
const SearchInput: React.FC<PropTypes.InferProps<typeof propTypes> &
  Omit<
    React.HTMLProps<HTMLInputElement>,
    'onChange' | 'value' | 'size'
  >> = props => {
// ....
```

Reply:
it worked! :awthanks: